io.github.jonmagon.kdiskmark.yaml

```
@@ -0,0 +1,43 @@
+app-id: io.github.jonmagon.kdiskmark
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+rename-desktop-file: kdiskmark.desktop
+rename-icon: kdiskmark
+copy-icon: true
+command: kdiskmark
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=host
+  - --device=dri
+modules:
+  - name: kdiskmark
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
```

```suggestion
      - -DCMAKE_BUILD_TYPE=RelWithDebInfo
```

This will allow to have debug symbols.